Manuals/Tools/Altibase_trunk/eng/Replication Manager User's Manual.md

```diff
@@ -335,17 +363,128 @@ It will sometimes be necessary to edit database connection information. For exam
 This menu is available regard of you are connected to the database or not. Select the database you wish to register or delete the Extra Host IP. Right-click on the database to open the context menu and select the "Manage Extra Host IP" item.
 
-##### Removing a Database Connection
+#### Removing a Database Connection
 
 When a registered database is no longer in use, you will probably want to remove it from the DB Connections panel. To remove a database, select the target database connection. Open the context menu by right-clicking on the database connection to be removed, and then left-click on "Remove" in the context menu.
 
-### Detailed Usage: Embedded Help
+### Working with Replication
+
+This section is intended to describe the detail usage of Replication Manager. It also describes the relationship between panes and objects.
+
+Replication Manager takes care of a single base model, which simulates databases and replication-related objects, while providing a various ways of visualizing and manipulating the model through three editable panes. The model is consists of six types of objects as follows:
+
+1. Replication Object: It is an object created with the CREATE REPLICATION statement and the common object among panes at Replication Manager. Please refer to "Understanding User Interface" for details.
+2. Replication Target Table Object: This is a table that is designated, using the CREATE REPLICATION or ALTER REPLICATION statement, to be replicated between corresponding replication nodes.
+3. DB Connections Object: It is the root object of "DB Connection" object to manage DB Connection. Also, it is the entry point for "DB Connections" pane.
+4. DB Connection Object: It is a representation of a connection to Altibase DBMS. Please refer to "Understanding User Interface" for details.
+5. Replication Pairs Object: It is the root object of "Replication Pair" and the entry point for "Replication Pairs" pane.
+6. Replication Pair Object: It is a pair of corresponding replication objects having the same name, one residing on each of two different nodes. Please refer to "Understanding User Interface" for details.
+
+Some objects in the model are show up few panes for the ease use of Replication Manager.
+
+Even though some objects shared by panes are the same ones, they may provide different functionalities according to the pane. For instance, Replication Object at DB Connections pane provides "Edit Table List" function, but Replication Object at Map pane doesn't. The reason of different functionalities on the same object according to the panes is to align functionalities to its context. At the previous example, editing replication target table list is quite intuitive at the database-centric view, DB Connections pane. On the while, the Map pane, which is intended to give brief overview, is not the right place to edit replication target table list.
```

Comment:
The reason of different functionalities on the same object according to the panes is to align functionalities to its context.
-> This is to align functionalities to its context.

Reply:
At the previous example, editing replication target table list is quite intuitive at the database-centric view, DB Connections pane. On the while, the Map pane, which is intended to give brief overview, is not the right place to edit replication target table list.
-> In this example, it is more reasonable to edit the replication target table list at the database-centric view, DB Connections pane because the Map pane is intended to give a brief overview.

Reply:
반영했습니다.